src/control/directive.ts

```diff
@@ -86,6 +88,10 @@ export class NgrxFormControlDirective<TStateValue extends FormControlValueTypes,
 
   @Input() ngrxValueConverter: NgrxValueConverter<TViewValue, TStateValue> = NgrxValueConverters.default<any>();
 
+  @Input() ngrxSuppressGlobalActionEmit: boolean = false;
```

Comment:
minor thing but the type annotation is redundant. i'd remove it for consistency with the rest of the code